data/policies/permissions.cas

```diff
@@ -14,3 +14,7 @@ domain foo {
 }
 
 resource read {}
+
+let some_perms = [ write read getattr ];
+
+let some_perms_other_order = [ read write setattr ];
```

Comment:
Did you intend for these to be used in this cas file or just make sure they don't break compile?

Reply:
I was just making sure they don't break compile.  It would probably make sense to use them as well though so that the test validates that they were actually processed correctly.